hack/test/overlays/test-kind/metrics/additional.jsonl

```diff
@@ -1 +1,3 @@
+{"Name":"kubernetes.controlplane.coredns.dns.request.duration.seconds.bucket","Tags":{"source":"","cluster":"","le":"","namespace":"","pod":"","server":"","type":"","zone":""}}
```

Comment:
Need clarification before porting this change PR #366: Does having coredns metrics in the hack/test/files/metrics.jsonl break the test in gke and eks? Is that why it is put explicitly in kind's additional metrics?